ignite/pkg/events/events_test.go

```diff
@@ -1,48 +1,56 @@
-package events
+package events_test
 
 import (
 	"testing"
 
 	"github.com/gookit/color"
+	"github.com/ignite-hq/cli/ignite/pkg/events"
 	"github.com/stretchr/testify/require"
 )
 
 func TestBusSend(t *testing.T) {
 	tests := []struct {
 		name  string
-		bus   Bus
-		event Event
+		bus   events.Bus
+		event events.Event
 	}{
 		{
 			name: "send status ongoing event",
-			bus:  make(Bus),
-			event: Event{
-				Status:      StatusOngoing,
+			bus:  events.NewBus(),
+			event: events.Event{
```

Comment:
@ivanovpetr can you please create an issue so we can update these tests to use NewOngoing and variants instead of events.Event?

Reply:
https://github.com/ignite-hq/cli/issues/2443